app/bundles/EmailBundle/Event/EmailSendEvent.php

```diff
@@ -45,6 +45,14 @@ class EmailSendEvent extends CommonEvent
 
     private array $textHeaders = [];
 
+    private bool $fatal = false;
+
+    private bool $skip = false;
+    /**
```

Comment:
missing a line b/w line number 50  and 51
```suggestion
    private bool $skip = false;
    
    /**
```

Reply:
Thank you, it's done.